index.js

```diff
@@ -12,3 +14,36 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+async function fetchCharacters() {
+  try {
+    const response = await fetch("https://rickandmortyapi.com/api/character");
+    if (response.ok) {
+      const data = await response.json();
+      const results = data.results;
+      cardContainer.innerHTML = "";
+      results.forEach((person) => {
+        const name = person.name;
+        const status = person.status;
+        const type = person.type;
+        const occurrences = person.episode.length;
+        const src = person.image;
+        const newCard = createCharacterCard(
+          name,
+          status,
+          type,
+          occurrences,
+          src
+        );
+        cardContainer.append(newCard);
+      });
+      return data;
```

Comment:
As you are already using your data to create cards, you do not need to return your data here.
```suggestion

```